packages/semver/src/builders/version/utils/try-bump.ts

```diff
@@ -32,47 +36,82 @@ export function tryBump({
     })
   );
 
-  const commits$ = version$.pipe(
-    switchMap((version) =>
+  const sinceGitRef$ = since$.pipe(
+    /** If since equals 0.0.0 it means no tag exist,
+      * then get the first commit ref to compute the initial version. */
+    switchMap((since) =>
+      iif(() => since === `0.0.0`, getFirstCommitRef(), of(since))
+    )
```

Comment:
When the current version is not found, maybe `since`'s value should be `null`. This way we can make the difference between detecting a `0.0.0` version and not detecting any version. 

If `since` is `null` (or currently `0.0.0`), then before bumping, we can log something like: _Couldn't find a previous version tag for this project. New version will be calculated based on all changes since first commit. If your project is already versioned, please tag the last release commit with `my-prefix-x-y-z` and run this command again._